parse_arguments.sh

```diff
@@ -117,4 +121,4 @@ export SKIP_IMPORT
 export SKIP_NETWORK_IMPORT
 export SKIP_REPLACE
 export PARALLEL_IMPORT
```

Comment:
Please review here. There should have been a export PROPERTIES_FILE in order for read_properties.sh file to override mirror_db.sh PROPERTIES_FILE variable.